docs/libraries/stream/reference/index.mdx

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
is this `docs/libraries/stream/reference` duplicate with `docs/libraries/streams/reference`?